internal/search/main.go

```diff
@@ -15,16 +15,18 @@ import (
 	"sync"
 	"time"
 
-	"github.com/rundsk/dsk/internal/ddt"
 	"github.com/blevesearch/bleve"
 	"github.com/blevesearch/bleve/analysis/analyzer/keyword"
 	"github.com/blevesearch/bleve/analysis/analyzer/simple"
 	"github.com/blevesearch/bleve/analysis/lang/de"
 	"github.com/blevesearch/bleve/analysis/lang/en"
 	"github.com/blevesearch/bleve/mapping"
 	"github.com/blevesearch/bleve/search/query"
+	"github.com/rundsk/dsk/internal/ddt"
 )
 
+const FILTER_RESULT_LIMIT = 500
```

Comment:
Go does constant names a little different, see https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const they follow the same rules as variables. Can you apply that small change?

```suggestion
const FilterResultLimit = 500
```

Reply:
ha, I was mislead by https://www.golangprograms.com/go-language/constants.html